static/app/components/tabs/index.spec.jsx

```diff
@@ -0,0 +1,203 @@
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
```

Comment:
nit: We should try to write new tests in typescript

Reply:
Didn't want to point this out... but yes... that would be cool! :)